pytest_django/plugin.py

```diff
@@ -231,7 +238,7 @@ def pytest_load_initial_conftests(early_config, parser, args):
 
         # Forcefully load django settings, throws ImportError or
         # ImproperlyConfigured if settings cannot be loaded.
-        from django.conf import settings
+        from django.conf import settings  # noqa
```

Comment:
Please go ahead :) It's fine as its own PR, I can fixup the conflicts when I rebase this branch.